src/common/metrics/weekly_new_cases_per_100k.tsx

```diff
@@ -22,24 +22,27 @@ export const WeeklyNewCasesPer100kMetric: MetricDefinition = {
   metricNameForSummaryStat: 'Weekly new cases',
 };
 
+const LIMIT_LOW = 200;
+const LIMIT_MEDIUM = Infinity;
+
```

Comment:
I think the _weekly new cases_ metric has only Low and High. In https://covid-projections-git-casulin-new-chart-week-5068d7-covidactnow.vercel.app/us/california-ca/county/plumas_county/?s=31701993, the color on the dot for weekly new cases is yellow (it should be dark orange), maybe we need to remove medium here?

<img width="895" alt="image" src="https://user-images.githubusercontent.com/114084/162790047-043f6886-708f-4dbd-9671-70b797bf40c3.png">